src/Symfony/Component/Routing/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -13,8 +13,8 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Config\FileLocator;
-use Symfony\Component\Routing\Loader\YamlFileLoader;
```

Comment:
please keep original ordering (that's our policy: alpha order for new items, "as is" for existing ones)